# Potato_sRNA_analysis
small RNA analyses [S. tuberosum]

```
input

scripts

output
```
tree structure


Read more about scripts in <p style="color:blue">./scripts/README.md</p>


<h4><u>Recommended sRNA data analysis pipeline / accompanying directories:</u></h4>

  - **Pre-processing** # Known miRNAs identification

  - **Novel_miRNA_analysis** # Novel miRNAs identification # Assigning the predicted miRNAs to known or novel MIR genes and families

  - **isomiRID** # Identification of miRNA variants (i.e. isomiRs)

  - **phasiRNA_analysis** # Identification of PHAS loci and phasiRNAs 

  - **DEA** # Differential expression analysis

  - **Target_prediction** # sRNA target prediction


<h4><u>Useful bases:</h4>

- www.mirbase.org

- https://rnacentral.org/

- ftp://ftp.solgenomics.net

- http://solanaceae.plantbiology.msu.edu/


<h4><u>Useful tools:</h4>

- [The small RNA group software for analysis and processing of NGS datasets](http://www.smallrnagroup.uni-mainz.de/software.html)

- [UEA small RNA Workbench](http://srna-workbench.cmp.uea.ac.uk/)

- [The ViennaRNA Package](https://www.tbi.univie.ac.at/RNA/)

- [CD-HIT Suite](http://weizhong-lab.ucsd.edu/cdhit_suite/cgi-bin/index.cgi)

- [GenomeTools](http://genometools.org/)

- [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc) or [MultiQC](https://multiqc.info/)

- [FASTX-Toolkit](http://hannonlab.cshl.edu/fastx_toolkit/commandline.html)

- [SAMtools](www.htslib.org/doc/samtools.html)

- [bedtools](https://bedtools.readthedocs.io/en/latest/index.html)

- [Cutadapt](https://cutadapt.readthedocs.io/en/stable/)

- [Bowtie](http://bowtie-bio.sourceforge.net/manual.shtml) and [Bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)

- [Mike Axtells' CleaveLand4](https://github.com/MikeAxtell/CleaveLand4/) [Mike Axtells' ShortStack](https://github.com/MikeAxtell/ShortStack)

- [de Oliveiras' isomiRID](https://github.com/lfelipedeoliveira/isomiRID)
